addons/web/static/src/js/views/pivot/pivot_model.js

```diff
@@ -599,7 +599,7 @@ var PivotModel = AbstractModel.extend({
         });
 
         var index = 0;
-        var rowGroupBys = !_.isEmpty(this.data.groupedBy) ? this.data.groupedBy : this.initialRowGroupBys;
+        var rowGroupBys = !_.isEmpty(this.data.groupedBy) ? this.data.groupedBy : _.clone(this.initialRowGroupBys);
```

Comment:
you can use slice() instead of _.clone (please always use native API when possible)